Dockerfile

```diff
@@ -20,7 +20,7 @@ COPY emails emails
 ENV NODE_ENV production
 RUN yarn build
 
-FROM golang:1.19.4-alpine3.15 as go-builder
+FROM golang:1.19.8-alpine3.17 as go-builder
```

Comment:
Untested assumption that we need to bump the alpine version here to get the more recent Go version.